### ISSUE TITLE
Windows PowerShell eating all the ram

### ISSUE THREAD

**Issue by David-Francesch** [oliverschwendener/ueli#602]
Since i have ueli installed a process in the task manager is showing and consumming all the ram, when it cant consume more it stops itself but a couple of minutes later its back again. I've been looking and touching around in the settings and reinstalled a couple of times and nothing changes.

![image](https://user-images.githubusercontent.com/36375397/102057002-8900b300-3ded-11eb-8784-b506f64dc7bf.png)

**Comment by tkohlmeier** [comment]
Could you check the command line of this powershell process? Just right
click on a column header and select command line. That would help narrowing
it down to the problematic plugin.

**Comment by David-Francesch** [comment]
Sure! i've tried to copy it but it is a gigantic line of code so i had to take screenshots, i hope it works for you, if not, i will copy it no problem.

These are the screenshots:
![image](https://user-images.githubusercontent.com/36375397/102063897-e9e0b900-3df6-11eb-953a-e45942b671a6.png)
![image](https://user-images.githubusercontent.com/36375397/102063910-ed744000-3df6-11eb-937b-69737ca5c003.png)
![image](https://user-images.githubusercontent.com/36375397/102063920-efd69a00-3df6-11eb-96fa-ea942fe06869.png)
![image](https://user-images.githubusercontent.com/36375397/102063928-f1a05d80-3df6-11eb-86ca-bf8798698ed0.png)
then, after the last screeshot follow these --> If($newStartApp

and thats all, if you need something else just let me know i will be in touch.

**Comment by David-Francesch** [comment]
![image](https://user-images.githubusercontent.com/36375397/102064145-40e68e00-3df7-11eb-8fcb-b6eedd9cd62b.png)

i just saw the images and i can't see a thing so i upload this one

**Comment by tkohlmeier** [comment]
The UWP plugin seems to be causing this. So that process should be gone if
you disable the UWP plugin. I hope I find some time to look into this.

**Comment by David-Francesch** [comment]
you're right, disabling the UWP option stops the process, i hope you can fix this asap, btw i love your app, the best one i've seen. Keep working on it because it's amazing.

**Comment by tkohlmeier** [comment]
I created a powershell script with some debugging outputs: [getUwpApps.zip](https://github.com/oliverschwendener/ueli/files/5730292/getUwpApps.zip)

Please extract this, open a command prompt, go to the location where you extracted the file to and run the script using this command: `powershell .\getUwpApps.ps1`

If you are not allowed to run the script because it is not signed, try this: `powershell -executionpolicy bypass -File .\getUwpApps.ps1`

Please post the output here. You can remove the app ids for privacy reasons if you like. And please monitor the memory consumption while the script is executing.

**Comment by televators** [comment]
So glad I found this. I was up until 3AM running recovery disks, emergency malware scanning, and all kinds of stuff because I turned on my computer and within 30 seconds had a 'low system memory' warning. Since I have 32GB and Windows had literally just started, I freaked out and saw a hundred powershell instances in task manager and then before I could do anything or even figure out what the hell was happening, system froze completely. Glad to see I don't have some wacky malware for the first time in a decade.

I've got Ueli installed on my Windows 7 rig (I also have Everything installed) so I can mostly mirror my work laptop with 10 + PowerToys. I'm going to try out the suggestions in this thread and report back for posterity.
